### PR TITLE
Correct HTML code when reporting `≤` and `≥`

### LINKED ISSUE #10046: HTML file using ≤ symbol instead of &le; in AnnualThermalResilienceSummary
Issue overview
--------------
Run any file that creates a AnnualThermalResilienceSummary report (I used the 5ZoneEconomicsTariffAndLifeCycleCosts.idf) and look for the Annual Thermal Resilience Summary report for the like showing "Safe" it uses the "≤" symbol instead of the "& le ;" 

```
<p><a href="#toc" style="float: right">Table of Contents</a></p>
<a name=AnnualThermalResilienceSummary::EntireFacility></a>
<p>Report:<b> Annual Thermal Resilience Summary</b></p>
<p>For:<b> Entire Facility</b></p>
<p>Timestamp: <b>2023-05-26
    08:30:41</b></p>
<b>Heat Index Hours</b><br><br>
<!-- FullName:Annual Thermal Resilience Summary_Entire Facility_Heat Index Hours-->
<table border="1" cellpadding="4" cellspacing="0">
  <tr><td></td>
    <td align="right">Safe (≤ 26.7&deg;C) [hr]</td>
    <td align="right">Caution (&gt; 26.7&deg;C, ≤ 32.2&deg;C) [hr]</td>
    <td align="right">Extreme Caution (&gt; 32.2&deg;C, ≤ 39.4&deg;C) [hr]</td>
    <td align="right">Danger (&gt; 39.4&deg;C, ≤ 51.7&deg;C) [hr]</td>
    <td align="right">Extreme Danger (&gt; 51.7&deg;C) [hr]</td>
  </tr>
```

This causes a problem with parsers that use the HTML file. Most browsers will display this just fine.

### Details
Some additional details for this issue (if relevant):
 - Windows
 - 23.1

### Checklist
Add to this list or remove from it as applicable.  This is a simple templated set of guidelines.
 - [ ] Defect file added (list location of defect file here)
 - [ ] Ticket added to Pivotal for defect (development team task)
 - [ ] Pull request created (the pull request will have additional tasks related to reviewing changes that fix this defect)

### PR DESCRIPTION
**Pull request by lymereJ**
Pull request overview
---------------------
 - Fixes #10046 

### Pull Request Author
Add to this list or remove from it as applicable.  This is a simple templated set of guidelines.
 - [x] Title of PR should be user-synopsis style (clearly understandable in a standalone changelog context)
 - [x] Label the PR with at least one of: Defect, Refactoring, NewFeature, Performance, and/or DoNoPublish
 - [x] Pull requests that impact EnergyPlus code must also include unit tests to cover enhancement or defect repair
 - [x] Author should provide a "walkthrough" of relevant code changes using a GitHub code review comment process
 - [x] If any diffs are expected, author must demonstrate they are justified using plots and descriptions
 - [x] If changes fix a defect, the fix should be demonstrated in plots and descriptions

### Reviewer
This will not be exhaustively relevant to every PR.
 - [ ] Perform a Code Review on GitHub
 - [ ] If branch is behind develop, merge develop and build locally to check for side effects of the merge
 - [ ] If defect, verify by running develop branch and reproducing defect, then running PR and reproducing fix
 - [ ] If feature, test running new feature, try creative ways to break it
 - [ ] CI status: all green or justified
 - [ ] Check that performance is not impacted (CI Linux results include performance check)
 - [ ] Run Unit Test(s) locally
 - [ ] Check any new function arguments for performance impacts
 - [ ] Verify IDF naming conventions and styles, memos and notes and defaults
 - [ ] If new idf included, locally check the err file and other outputs

### CODE REVIEW

File: src/EnergyPlus/OutputReportTabular.cc
Comment (lymereJ) [inline src/EnergyPlus/OutputReportTabular.cc]: `≥` and `≤` are not ASCII characters, so we look at the next few characters and see if the series corresponds to `≥` and `≤`.

**Comment by lymereJ** [comment]
Example of HTML output generated with this branch:

image

image

**Comment by dareumnam** [comment]
I built it locally and checked that the HTML output uses `&le;` instead of `≤`. CI seems happy here.

File: src/EnergyPlus/OutputReportTabular.cc
Comment (Myoldmopar) [reply src/EnergyPlus/OutputReportTabular.cc]: ... negative numbers into the char constructor look funny to me.  Maybe it's just me.  What's the reason behind them?

File: tst/EnergyPlus/unit/OutputReportTabular.unit.cc
Comment (Myoldmopar) [inline tst/EnergyPlus/unit/OutputReportTabular.unit.cc:665]: Obviously this is working thanks to this test, I'd just like a comment back about the negative numbers in char above.

**Review by Myoldmopar — COMMENTED** [review]
I"m curious about the negative numbers in the char constructor, so I'll hold this as just a comment for now.  Once that's resolved, I'll approve and merge.

**Comment by Myoldmopar** [comment]
Thanks for confirming the fix @dareumnam, this one is definitely close to merging, just one thing to discuss.

**Comment by lymereJ** [comment]
@Myoldmopar - I swapped the negative values to the corresponding positive ones.

While looking for a value to pass to `char` (since they're not ASCII characters) I found the original values I used [here](https://gsamaras.wordpress.com/code/negative-ascii-codes/). I did find positive values in the extended ASCII character table for them so I swapped the values.

File: src/EnergyPlus/OutputReportTabular.cc
Comment (Myoldmopar) [inline src/EnergyPlus/OutputReportTabular.cc:17336]: Thanks for changing this; I realize it's technically the same thing, but this is one less thing to grok when looking at this code.

**Review by Myoldmopar — APPROVED** [review]
All good now, and I tested locally with latest develop and it passes happily.  This is merging.  Thanks @lymereJ

**Pull request merged**
